apps/emqx_ldap/src/emqx_ldap.erl

```diff
@@ -142,7 +140,7 @@ on_start(
     ?SLOG(info, #{
         msg => "starting_ldap_connector",
         connector => InstId,
-        config => emqx_utils:redact(Config)
```

Comment:
In the particular case of LDAP schema, for example, there seems to be this `bind_password` field that _could_ be lurking around in the config.  

I haven't checked if it's really the case here, but I think it's safer to keep those `redact` calls wherever we log configurations.

Reply:
Besides, it'll format passwords as `******` rather than `#Fun<mod.43.3316493>`.